code/game/machinery/computer/hybrisa_slotmachine.dm

```diff
@@ -0,0 +1,339 @@
+/*******************************\
+|   Slot Machines |
+|   Original code by Glloyd |
+|   CM port by Cuberound    |
+\*******************************/
+
+#define SPIN_PRICE 10
+#define SMALL_PRIZE 100
+#define BIG_PRIZE 500
+#define JACKPOT 5000
+#define SPIN_TIME 65 //As always, deciseconds.
+#define REEL_DEACTIVATE_DELAY 7
+#define JACKPOT_SEVENS "space-shuttle"
+#define FA_ICON_LEMON "lemon"
+#define FA_ICON_STAR "star"
+#define FA_ICON_BOMB "bomb"
+#define FA_ICON_BIOHAZARD "biohazard"
+#define FA_ICON_APPLE_WHOLE "apple-alt"
+#define FA_ICON_7 "space-shuttle"
+#define FA_ICON_DOLLAR_SIGN "fas fa-dollar-sign"
+
+/obj/structure/machinery/computer/hybrisa/misc/slotmachine
+	name = "slot machine"
+	desc = "A slot machine."
+	icon = 'icons/obj/structures/props/furniture/slot_machines.dmi'
+	icon_state = "slotmachine"
+	bound_width = 32
+	bound_height = 32
+	anchored = TRUE
+	density = TRUE
+	layer = WINDOW_LAYER
```

Comment:
Is there a way to layer them so if you're standing on the tile above it would be layered above you? They're offset, but it would still clip, since they aren't 32x32. I just don't really know how to layer these properly since they are long.

Reply:
I would just not pixel offset them any different than is shown so that marines are below them aren't overlapping and when behind you just have them layered above the mob. As far as making it so when you stand above something you appear above it, but otherwise you appear below it automatically: you can only do this is the layer exactly matches the mob, but xenos use a different layer than humans so there isn't a single layer to use unless it somehow is a situation where only humans or only xenos would be present.

Reply:
I could like make the prop in two parts, but then again it wouldn't look right, otherwise I'll just undo the offset, but there's still gonna be some clipping since they're not 32x32 and don't fit on the tile exactly.

Reply:
See how you like it with it on the xeno mob layer once I finish changes (meaning xenos will automatically be above or below since it matches their layer, but for humans its above them when they're above it since when they're below it they don't overlap with it): 
![image](https://github.com/user-attachments/assets/772bacdd-1fe6-4169-b4fb-fc26015788d3).

Reply:
Yeah that works well. What layer is it? BIG_XENO_LAYER or something?

Reply:
Yes